app/views/external_users/advocates/hardship_claims/_basic_fees_form_step.html.haml

```diff
@@ -0,0 +1,10 @@
+- locale_scope = 'external_users.hardship_claims.basic_fees.fields'
+
+= content_for :page_title, flush: true do
+  = t('.page_title')
+
+= render partial: 'external_users/claims/fees_shared_header', locals: {page_header: t('page_header', scope: locale_scope), page_hint: t('page_hint', scope: locale_scope), fees_calculator_html: nil}
```

Comment:
Note: line seems really long.

Reply:
https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3258#discussion_r412930434